scripts/migrations/migrations.sh

```diff
@@ -1,30 +1,37 @@
-function run_migrations() {
-  echo "Starting migrations..."
-  echo
+function version_greater_than() {
+    [[ "$1" = "$(echo -e "$1\n$2" | sort -V | tail -n 1)" ]]
+}
 
-  euclid_field_version=$(jq -r '.version // empty' "$ROOT_PATH/euclid.json")
+function run_migrations() {
+  echo_title "Starting migrations..."
+  echo_white
+  
   VERSION=$1
-
+  VERSION_WITHOUT_V="${VERSION#v}"
   cd $INFRA_PATH
 
-  if [ ! -d "euclid-development-environment" ]; then
-    git clone https://github.com/Constellation-Labs/euclid-development-environment.git
-  fi
-
-  cp $ROOT_PATH/euclid.json $ROOT_PATH/old-euclid.json
 
-  if [ -z "$euclid_field_version" ]; then
+  if version_greater_than $VERSION_WITHOUT_V "0.9.0"; then
```

Comment:
Where is the version number coming from? It's coming from euclid.json right? We'll also need to handle the pre-0.9.0 case where euclid.json didn't have a version field.

Reply:
Also, this will run for 10.x.x also right? We only want it to run if it hasn't already run so it should run if the euclid.json version is greater than 0.9.0 and less than 0.10.0 in this case.

Reply:
Yes it will run sequentially, in this case, it will run if we provide 0.10.0 version too